plutus-core/plutus-core/src/PlutusCore/ParserCommon.hs

```diff
@@ -0,0 +1,179 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
nit: we usually have a block of `LANGUAGE` pragmas and any `OPTIONS_GHC` in a separate block below that.